src/utils/responsiveness.cljs

```diff
@@ -0,0 +1,10 @@
+(ns utils.responsiveness)
+
+(def ^:const IPHONE_11_PRO_VIEWPORT_WIDTH 375)
```

Comment:
I don't think a specific device width needs to be in the calculation even if that's what we are building to. 

We just need the current device's width, and each circle's width, and then we can calculate if a part of the last circle will be visible or not

Reply:
We still don't need iphone 11 width. On figma we have 6 and a half circles visible, and 7 spaces for margin. If the device width is `X` and cirlce's width is `Y`, then we have the total margin to be `X - (6.5 Y)`, let's call that value `Z`. Therefore, one margin is `Z / 7`.

Reply:
That'd work too, But I still think having a reference to Figma's dimension is a better approach.